tests/sync/test_admin_status.py

```
@@ -158,6 +158,105 @@ def test_fax_queue_counts(self, mock_models, mock_actors, mock_fax):
+    status = _FakeStatus()
+    start_time = timezone.now() - datetime.timedelta(seconds=42)
+    close_time = timezone.now()
+
+
+class _FakeTemporalClient:
+    """Stands in for a temporalio Client: two completed runs, one listed.
+
+    Records every visibility query so tests can check their shape.
+    """
+
+    queries: list = []
```

Ruff (0.16.2)

[warning] Mutable default value for class attribute

(RUF012)